packages/ui/src/editor/editor.css

```
@@ -0,0 +1,46 @@
+}
+
+.quill-wrapper .ql-editor.ql-blank::before {
+  color: #9e9e9e; /* gray-500 */
+  font-style: normal;
+  left: 12px;
+}
+
+/* 포커스 스타일 */
+.quill-wrapper:has(.ql-editor:focus) {
+  outline: none;
+  ring: 2px;
```

Stylelint (17.5.0)

[error] Unexpected unknown property "ring" (property-no-unknown)

(property-no-unknown)